docs/reference/auditbeat/setup-repositories.md

```diff
@@ -17,10 +17,111 @@ to sign all our packages. It is available from [https://pgp.mit.edu](https://pgp
 
 ## APT [_apt]
 
-Version 9.0.0-beta1 of Beats has not yet been released.
+To add the Beats repository for APT:
+
+1. Download and install the Public Signing Key:
+
+    ```shell
+    wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | sudo apt-key add -
+    ```
+
+2. You may need to install the `apt-transport-https` package on Debian before proceeding:
+
+    ```shell
+    sudo apt-get install apt-transport-https
+    ```
+
+3. Save the repository definition to  +/etc/apt/sources.list.d/elastic-{{major-version}}.list+:
+
+    ```shell subs=true
+    echo "deb https://artifacts.elastic.co/packages/{{major-version}}/apt stable main" | sudo tee -a /etc/apt/sources.list.d/elastic-{{major-version}}.list
+    ```
+
+    :::{note}
+    The package is free to use under the Elastic license. An alternative package which contains only features that are available under the Apache 2.0 license is also available. To install it, use the following sources list:
+
+    ```shell subs=true
+    echo "deb https://artifacts.elastic.co/packages/oss-{{major-version}}/apt stable main" | sudo tee -a /etc/apt/sources.list.d/elastic-{{major-version}}.list
+    ```
+    :::
+
+    :::{warning}
+    To add the Elastic repository, make sure that you use the `echo` method  shown in the example. Do not use `add-apt-repository` because it will add a `deb-src` entry, but we do not provide a source package.
```

Comment:
```suggestion
    To add the Elastic repository, make sure that you use the `echo` method shown in the example. Do not use `add-apt-repository` because it will add a `deb-src` entry, but we do not provide a source package.
```